unit-tests/sw-dev/sw.py

```diff
@@ -0,0 +1,103 @@
+# License: Apache 2.0. See LICENSE file in root directory.
+# Copyright(c) 2023 Intel Corporation. All Rights Reserved.
+
+import pyrealsense2 as rs
+from rspy import log, test
+from time import time
+
+
+fps = 30
+w = 640
+h = 480
+bpp = 2  # bytes
+pixels = bytearray( b'\x00' * ( w * h * 2 ))  # Dummy data
```

Comment:
better to use `bpp`s value instead of `2`. Does this support formats like Y8 or RGB8?

Reply:
Right, but this was copy-paste from sw.py inside `unit-tests/syncer/`. It should be able to support any format (like the DDS frames) but right now no content is actually written there...

Reply:
Done